_posts/en/newsletters/2025-11-28-newsletter.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Bitcoin Optech Newsletter #382'
+permalink: /en/newsletters/2025/11/28/
+name: 2025-11-28-newsletter
+slug: 2025-11-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Stats on compact block reconstructions updates:** 0xB10C posted to
+  [Delving Bitcoin][0xb10c delving] about his stats on compact block
+  reconstruction. This is an update to a previous [optech
+  post][compact block optech], since then 0xB10C has provided three updates
+  on compact block reconstruction. He has also tracking the average size of
+  requested transactions in kB, because of the [previous
+  post][david delving post] by Gumberg. The new chart is similar looking as
+  the share of compact block reconstructions without request transactions
+  chart and can be found directly under them in each post.
+
+  0xB10C updated one of his nodes named bob to the master branch and found
+  that because there were no existing sub-1 sat/vbyte transactions there was
+  no improvment for the first few days, afterwards block reconstruction rates
+  improved significantly. We could also see an improvement in the average
+  size of reqeuested transactions in kB.
+
+  In the [third update][0xb10c thrid update] he had posted, 0xB10C mentions
+  that he swiched the nodes to run with a lowered minrelayfee. This caused
+  most the nodes to have better reconstruction rate and less data requested
+  from peers. He also mentions in hindsight it would have been better to not
+  have updated all the nodes and to have kept some on v29. This would have
+  made it easier to compare between node versions and settings.
+
+  Overall updating to the latest version of Bitcoin Core and dropping the
+  minrelayfee has improved block reconstruction rates and lowered the amount
+  of data requested from peers.
+
+- **Motion to activate BIP3**: Murch [posted][murch ml] to the Bitcoin-Dev
+  Mailing List a formal motion to activate [BIP3][]. The goal of this
+  improvement proposal is to provide new guidelines for the preparation of new
+  BIPS, thus replacing [BIP2][].
+
+  According to the author, the proposal, which has been in "Proposed" status for
+  more than 7 months, has no unaddressed objections and a growing list of
+  contributors left an ACK on [BIPs #1820][]. Thus, following the procedure
+  expressed by BIP2 for activating Process BIPs, he conceded 4 weeks, untill
+  2025-12-02, to evaluate the proposal, ACK the PR or to state concerns and
+  raise objections. After that date BIP3 will replace BIP2 as BIPs Process.
+
+  Some small objections have been raised in the thread, mainly related to the
+  use of AI/LLM tools in the BIPs submission process, which are currently being
```

Comment:
Perhaps reference this recent PR that we covered

https://bitcoinops.org/en/newsletters/2025/10/31/#bips-2006